### PR TITLE
Support for mutiple class names as key

### PR DESCRIPTION
**Pull request by phanan**
This is to replace #2581.

### CODE REVIEW

File: src/directives/internal/class.js [hunk 1]
```diff
@@ -63,9 +63,11 @@ function setObjectClasses (el, obj) {
   var keys = Object.keys(obj)
   for (var i = 0, l = keys.length; i < l; i++) {
     var key = keys[i]
-    if (obj[key]) {
-      addClass(el, key)
-    }
+    if (!obj[key]) continue
+
+    key.split(' ').forEach(function (k) {
```
Comment (yyx990803) [inline src/directives/internal/class.js]: Unconditional split + forEach can be slow, and class bindings can be a hot spot so performance is important.

Let's do an `key.indexOf(' ') > 1` check first, then use a normal for loop.

Comment (phanan) [inline src/directives/internal/class.js]: That's what concerned me too, though I don't have an exact idea about the speed comparison between the check and a blind `split`.

File: src/directives/internal/class.js [hunk 2]
```diff
@@ -77,3 +73,39 @@ function stringToObject (value) {
...
+
+/**
+ * Add or remove a class/classes on an element
+ *
+ * @param {Element} el
+ * @param {String} key The class name. This may or may not
+ *                     contain a space character, in such a
+ *                     case we'll deal with multiple class
+ *                     names at once.
+ * @param {Boolean} add
```
Comment (yyx990803) [inline src/directives/internal/class.js]: instead of passing a boolean, we can just pass in `addClass` or `removeClass` and call it directly.

Comment (phanan) [inline src/directives/internal/class.js]: Great idea. Will do.

**Pull request merged**